src/it/streamingcommunity/src/eu/kanade/tachiyomi/animeextension/it/streamingcommunity/StreamingCommunity.kt

```diff
@@ -252,68 +253,83 @@ class StreamingCommunity : ConfigurableAnimeSource, AnimeHttpSource() {
 
     // ============================ Video Links =============================
 
-    override fun videoListRequest(episode: SEpisode): Request = GET("$baseUrl/watch/${episode.url}", headers)
+    interface VideoProvider {
+        fun getVideoList(): List<Video>
+    }
```

Comment:
What's the point of this?

Reply:
I was testing something and I missed deleting it